lms/djangoapps/grades/new/course_grade.py

```diff
@@ -114,7 +111,7 @@ def summary(self):
         grade_summary['totaled_scores'] = self.subsection_grade_totals_by_format
         grade_summary['raw_scores'] = list(self.locations_to_weighted_scores.itervalues())
 
-        self._log_event(log.warning, u"grade_summary, percent: {0}, grade: {1}".format(self.percent, self.letter_grade))
+        self._log_event(log.info, u"grade_summary, percent: {0}, grade: {1}".format(self.percent, self.letter_grade))
```

Comment:
Can we keep a running log of the grade "percentage" updates somewhere, for now?  At least until the Grades Eventing work is completed.  So either keep this as a `warning` for now, or add it to another log statement.  What do you think?